src/map/utils/charutils.cpp

```diff
@@ -2724,6 +2724,9 @@ namespace charutils
         if ((PChar->WorkingSkills.rank[SkillID] != 0) && !(PChar->WorkingSkills.skill[SkillID] & 0x8000))
         {
             uint16 CurSkill = PChar->RealSkills.skill[SkillID];
+            uint16 CapSkill = battleutils::GetMaxSkill(SkillID, PChar->GetMJob(), PChar->GetMLevel());
+            // Max skill this mob level will allow.
```

Comment:
Would be clearer to say "victim" or "target" instead of "mob" here, that's how it's called from `attack.cpp` and `battleutils.cpp` etc.